graf3d/eve/src/TEveCalo3DGL.cxx

```diff
@@ -529,10 +529,7 @@ void TEveCalo3DGL::RenderEndCapCell(const TEveCaloData::CellGeom_t &cellData, Fl
 
    RenderBox(box);
 
-   if (z1 > 0)
-      offset += towerH * Cos(cellData.ThetaMin());
-   else
-      offset -= towerH * Cos(cellData.ThetaMin());
+   offset = towerH;
```

Comment:
@alja, here `offset = towerH;`, but in eve7 it is `offset += towerH`
Is it correct?